locustfile.py

```diff
@@ -192,8 +192,14 @@ def poll_upload_until_ready(self, url):
             with self.client.get(url, allow_redirects=False,
                                  name='/en-US/developers/upload/:uuid/json',
                                  catch_response=True) as response:
-                if response.status_code == 200:
+                try:
                     data = response.json()
+                except ValueError:
+                    return response.failure(
```

Comment:
is it necessary to catch this? Won't a ValueError show up in the exceptions field anyway? Or did it not? I have seen cases where a worker had a traceback but it didn't show up in stats for some odd reason.